src/Uno.UI/UI/Xaml/FrameworkElement.cs

```diff
@@ -527,6 +528,13 @@ async void ApplyPhase()
 			}
 		}
 #endif
+		/// <summary>
+		/// When true (normally because the IsUiAutomationMappingEnabled build property is set), setting the <see cref="Name"/> property
+		/// programmatically will also set the appropriate test identifier for Android/iOS. Disabled by default because it may interfere
+		/// with accessibility features in non-testing scenarios.
+		/// </summary>
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		public static bool IsUiAutomationMappingEnabled { get; set; } = false;
```

Comment:
Can you move that API here: https://github.com/unoplatform/uno/blob/8dcc6f2d3aa5d57fd3ee96efe5fe73855a45a0da/src/Uno.UI/UI/Xaml/FrameworkElementHelper.cs

You can keep the property here, but internal. It's going to help for compatibility management.

Reply:
done